chainbase/src/main/java/org/tron/core/db2/core/SnapshotManager.java

```diff
@@ -117,18 +116,6 @@ public void init() {
     exitThread.start();
   }
```

Comment:
Why not adjust exitThread to SingleThreadExecutor?

Reply:
`exitThread` will only run `once`, when it receives the exit signal ` LockSupport.unpark(exitThread) ` then `System.exit(1)`, otherwise it blocks in here `  LockSupport.park() `, unlike other single-thread that has while(true) logic.